projects/common/src/utilities/validators/form-validators.ts

```diff
@@ -0,0 +1,14 @@
+import { AbstractControl } from '@angular/forms';
+import { getStringsFromCommaSeparatedList } from '../formatters/string/string-formatter';
+import { areEmailAddressesValid } from './email-validator';
+
+export const validateCommaSeparatedEmails = (control: AbstractControl) => {
+  const emails = getStringsFromCommaSeparatedList(control.value);
+
+  // tslint:disable: no-null-keyword
+  return control.value.length === 0
+    ? { emptyEmails: true }
+    : !areEmailAddressesValid(emails)
+    ? { invalidEmails: true }
```

Comment:
IMO, having two different error types for the same validator is not a good idea. It is confusing to me. It will anyways give you an invalid state with the same object.